puppet/modules/vault_client/manifests/cert_service.pp

```diff
@@ -11,47 +11,49 @@
   Integer $key_bits = 2048,
   Integer $frequency = 86400,
   Array $exec_post = [],
+  Boolean $run_exec = true,
 )
 {
   require vault_client
 
   $service_name = "${name}-cert"
-
   exec { "${service_name}-systemctl-daemon-reload":
     command     => 'systemctl daemon-reload',
     refreshonly => true,
     path        => $::vault_client::path,
   }
+  if $run_exec {
```

Comment:
@JoshVanL I switched it now to encapsulating the code inside the run_exec. TBH I am not a big fan of this. I have to include a lot of other dependencies in it. I also had to do the same for  https://github.com/jetstack/tarmak/pull/390/files#diff-141eea4c2dc79f99116deed2572a20e2R38 because that also depends of this file. 

I prefer to go back to the `/bin/true` one. I know it looks weird but with the needed code comments it should be clear and clean enough. What do you think?

Reply:
Fair enough, yeah sure